Added DRT Strategies Inc GRC tool (#1122)
usnistgov/OSCAL: docs/content/tools/_index.md

| [Atlasity: Continuous Compliance Automation](https://atlasity.io/partnership/) | C2 Labs | Atlasity CE (release 2.0) runs in any environment and supports the development of OSCAL v1.0 content for Catalogs, Profiles, System Security Plans and Components. Additional detail can be found in this blog post: [Atlasity Delivers Free Tools to Create OSCAL Content](https://www.c2labs.com/post/atlasity-delivers-free-tools-to-create-oscal-content). | community edition |
| [control_freak](https://controlfreak.risk-redux.io/) | Risk Redux | This tool seeks to provide folks with a searchable and easy-to-navigate reference for NIST SP 800-53 Revision 5. It is [an open-source application from the Risk Redux project](https://github.com/risk-redux/control_freak), built using parsed content directly from the OSCAL repositories. | open-source |

> | [DRT Confidence](https://www.drtstrategies.com/drt-confidence-fedramp/) | DRT Strategies Inc. | DRTConfidence is a next generation Governance, Risk and Compliance (GRC) solution to help organizations transition to OSCAL and continuous compliance. DRTConfidence is available in a FedRAMP High environment, supports ALL OSCAL artifacts: Catalogs, Profiles, System Security Plans, Security Assessment Plans, Security Assessment Report, POAM and is compliant with OSCAL v1.0.0. Additional Information can be found at [DRT Confidence for FedRAMP](https://www.drtstrategies.com/drt-confidence-fedramp/). | Commercial License |

Reviewer: Very minor grammar nits around plurals and phrasing (there is no one true FedRAMP High Environment, but many possible environments, right?) and some other minor stuff.

Thanks for bringing it back up to HEAD and getting rid of the need for rebasing! :-)

```suggestion
| [DRT Confidence](https://www.drtstrategies.com/drt-confidence-fedramp/) | DRT Strategies Inc. | DRTConfidence is a next generation Governance, Risk and Compliance (GRC) solution to help organizations transition to OSCAL and continuous compliance. DRTConfidence is available in FedRAMP High environments and supports all OSCAL artifacts: Catalogs, Profiles, System Security Plans, Security Assessment Plans, Security Assessment Reports, POAMs and conforms to the OSCAL v1.0.0 specification and its schemas. Additional information can be found at [DRT Confidence for FedRAMP](https://www.drtstrategies.com/drt-confidence-fedramp/). | Commercial License |
```

> | [DRT Confidence](https://www.drtstrategies.com/drt-confidence-fedramp/) | DRT Strategies Inc. | DRTConfidence is a next generation Governance, Risk and Compliance (GRC) solution to help organizations transition to OSCAL and continuous compliance. DRTConfidence is available in FedRAMP High environments and supports all OSCAL artifacts: Catalogs, Profiles, System Security Plans, Security Assessment Plans, Security Assessment Reports, POAMs and conforms to the OSCAL v1.0.0 specification and its schemas. Additional information can be found at [DRT Confidence for FedRAMP](https://www.drtstrategies.com/drt-confidence-fedramp/). | Commercial License |

Reviewer 2: Is the Component Definition not supported or omitted by mistake? If not supported, maybe the statement should not read 'all OSCAL artifacts'. If omitted by mistake, please include it.

Author: It was omitted by mistake, we will add it and resubmit
